### ISSUE TITLE
nan is not defined

### ISSUE THREAD

**Issue by antgonza** [biocore/empress#126]
Hello,

I was trying to use Empress for one of my datasets and the page wasn't loading without any errors or messages; however, after a while I finally got these error in the Chrome console:
```
require-2.1.22.min.js:19 ReferenceError: nan is not defined
    at empress.html:129
    at Object.execCb (require-2.1.22.min.js:30)
    at ea.check (require-2.1.22.min.js:18)
    at ea.<anonymous> (require-2.1.22.min.js:23)
    at require-2.1.22.min.js:8
    at require-2.1.22.min.js:24
    at x (require-2.1.22.min.js:7)
    at ea.emit (require-2.1.22.min.js:24)
    at ea.check (require-2.1.22.min.js:20)
    at ea.<anonymous> (require-2.1.22.min.js:23)
```

Any thoughts on how to fix?

**Comment by kwcantrell** [comment]
@antgonza can you post the list of commands you used to run empress? I have not seen that error before so it would be helpful to know what commands were used.

**Comment by antgonza** [comment]
The command was: 
```bash
qiime empress plot \
   --i-tree insertion-tree.qza  \
   --i-feature-table feature-table.qza  \
   --m-sample-metadata-file metadata.txt  \
   --m-feature-metadata-file silva-taxonomy.qza  \
   --o-visualization empress.qzv
```

**Comment by ElDeveloper** [comment]
Resolved as part of #127
